index.js

```diff
@@ -308,8 +308,10 @@ module.exports = function StateProvider(makeRenderer, rootElement, stateRouterOp
 			parameters = extend(lastStateStartedActivating.get().parameters, parameters)
 		}
 
-		prototypalStateHolder.guaranteeAllStatesExist(stateName)
-		var route = prototypalStateHolder.buildFullStateRoute(stateName)
+		var destinationState = stateName === null ? lastStateStartedActivating.get().name : stateName
```

Comment:
We're sure that `lastStateStartedActivating.get()` will always return a state? What happens if in your code, you do a `state.go(null)` when you had no "previous state"?